src/components/confirm-dialog/ConfirmDialog.stories.ts

```diff
@@ -20,12 +20,12 @@ const Template = (args, { argTypes }) => ({
       await Vue.nextTick() // wait that mounted() finished
       vm.open(
         'Confirm Dialog Title',
-        'Confirm dialog descriptor, ask your confirmation question here.',
+        'Confirm dialog message. Ask your confirmation question here.',
         {
-          width: '45rem',
+          width: '30rem',
```

Comment:
Was 45 too big for the story window?

Reply:
No but I thought this looked better :)

![image](https://user-images.githubusercontent.com/10002889/159532098-ec466246-6dc3-4b2f-b46d-6fa6086efc8c.png)